modules/compose/templates/docker-compose-container.yml.erb

```diff
@@ -17,12 +30,12 @@ services:
 <% end -%>
       - '<%= scope.lookupvar('compose::codenvy_folder') -%>/config/haproxy/haproxy.cfg:/usr/local/etc/haproxy/haproxy.cfg'
       - '<%= scope.lookupvar('compose::codenvy_folder') -%>/config/haproxy/maintenance.html:/etc/haproxy/maintenance.html'
-      - '<%= scope.lookupvar('compose::codenvy_folder') -%>/logs/haproxy/:/var/log/haproxy/'
```

Comment:
haproxy send logs to rsyslog server `log         codenvy-rsyslog:514 local2` we don't need volume for that